server/service/src/backend_plugin/types/transform_requisition_lines.rs

```diff
@@ -0,0 +1,27 @@
+use crate::backend_plugin::{plugin_provider::PluginInstance, *};
+use plugin_provider::{call_plugin, PluginResult};
+use repository::{RequisitionLineRow, RequisitionRow};
+use serde::{Deserialize, Serialize};
+use ts_rs::TS;
+
+#[derive(TS, Clone, Deserialize, Serialize)]
+#[ts(rename = "TransformRequisitionLineInput")]
+pub struct Input {
+    pub requisition: RequisitionRow,
+    pub lines: Vec<RequisitionLineRow>,
+}
+
+#[derive(TS, Clone, Deserialize, Serialize)]
+#[ts(rename = "TransformRequisitionLineOutput")]
+pub struct Output {
+    pub transformed_lines: Vec<RequisitionLineRow>,
```

Comment:
Using transformed_lines rather then just Vec<RequisitionLineRow> because we would also need to expose pluginData, so that it's returned by the plugin, we would need to update suggested quantity based on the area AMC in some cases